cardano-crypto-tests/src/Test/Crypto/Util.hs

```diff
@@ -174,6 +204,20 @@ prop_raw_serialise serialise deserialise x =
       Just y  -> y === x
       Nothing -> property False
 
+prop_raw_deserialise :: 
```

Comment:
This name is a bit unintuitive - it suggests a "positive" property of deserialization, similar to how `prop_raw_serialize` tests a positive property of serialization; however, `prop_raw_deserialise` does not verify that deserialization succeeds when it should, but rather that it does not succeed in two specific scenarios (input too long, and input too short).